src/main/scala/org/hatdex/dex/api/services/DexOffers.scala

```diff
@@ -11,22 +11,24 @@ package org.hatdex.dex.api.services
 
 import java.util.UUID
 
-import org.hatdex.dex.api.json.DexJsonFormats
-import org.hatdex.dex.api.models.OfferClaimsInfo
+import scala.concurrent.{ ExecutionContext, Future }
+
+import io.dataswift.models.dex.OfferClaimsInfo
 import play.api.Logger
 import play.api.http.Status._
 import play.api.libs.ws._
 
-import scala.concurrent.{ ExecutionContext, Future }
-
 trait DexOffers {
   val logger: Logger
   val ws: WSClient
   val schema: String
   val dexAddress: String
 
-  import DexJsonFormats.offerClaimsInfoFormat
+  import io.dataswift.models.dex.json.DexJsonFormats._
 
+  @deprecated("This endpoint does not exist in dex",
+              since = "0.0.0"
+  ) //\todo remove this ? what does this affect ? is anyone using it ?
```

Comment:
This is a dinosaur known with many names. `Datatrader` was the latest one. Before that was known as [Databuyer](https://github.com/dataswift/DataBuyer)

Not sure on the state of it or any future plans. I am not sure if it's live somewhere even. We had some discussions with a university to spin something off based on Databuyer but that was years ago. So @augustinas should chip in here.

Reply:
i think you missed the point, this function calls ```/api/offer/$offerId/claims``` but that endpoint does not exist